cli.py

```diff
@@ -0,0 +1,273 @@
+        if args.names != '':
+            # if names file not given, assume id-based detection output
+            with open(args.names, 'r') as r:
+                names = list(map(str.strip, r.readlines()))
+                for det in det_anno:
+                    _out = names[int(det._class_id)]
+                    det._class_id = _out
+
+    # print out results of annotations loaded:
+    print("%d ground truth bounding boxes retrieved"%(len(gt_anno)))
+    print("%d detection bounding boxes retrieved"%(len(det_anno)))
+
+    # compute bboxes with given metric:
+
+    # COCO (101-POINT INTERPOLATION)
+    if args.metrics == 'coco':
+        logging.info("Running metric with COCO metric")
+
+        # use coco_out for PR graphs and coco_sum for just the AP
+        coco_sum = coco_evaluator.get_coco_summary(gt_anno, det_anno)
+        coco_out = coco_evaluator.get_coco_metrics(gt_anno, det_anno, iou_threshold=args.threshold)
+        
+        value_only = tuple([float(_i[1]) for _i in coco_sum.items()])
+        print( ('\nCOCO metric:\n'
+                'AP [.5:.05:.95]: %f\n'
+                'AP50: %f\n'
+                'AP75: %f\n'
+                'AP Small: %f\n'
+                'AP Medium: %f\n'
+                'AP Large: %f\n'
+                'AR1: %f\n'
+                'AR10: %f\n'
+                'AR100: %f\n'
+                'AR Small: %f\n'
+                'AR Medium: %f\n'
+                'AR Large: %f\n\n'%value_only) )
+
+        print("Class APs:")
+        for item in coco_out.items():
+            if item[1]['AP'] != None:
+                print("%s AP50: %f\n"%(item[0], item[1]['AP']))
+            else:
+                logging.warning('AP for %s is None'%(item[0]))
+
+        if args.prgraph:
+            plot_coco_pr_graph(coco_out, mAP=coco_sum['AP50'], ap50=coco_sum['AP'], savePath=args.savepath, showGraphic=False)
+        return coco_sum
+
+    # 11-POINT INTERPOLATION:
+    elif args.metrics == 'voc2007':
+        logging.info("Running metric with VOC2007 metric")
+        
+        voc_sum = pascal_voc_evaluator.get_pascalvoc_metrics(gt_anno, det_anno, iou_threshold=args.threshold, method=MethodAveragePrecision.ELEVEN_POINT_INTERPOLATION)
+        print("mAP: %f"%(voc_sum['mAP']))
+        print("Class APs:")
+        for class_item in voc_sum['per_class'].items():
+            if class_item[1]['AP'] != None:
+                print("%s AP: %f"%(class_item[0], class_item[1]['AP']))
+            else:
+                logging.warning('AP for %s is None'%(class_item[0]))
+
+        if args.prgraph:
+            pascal_voc_evaluator.plot_precision_recall_curve(voc_sum['per_class'], mAP=voc_sum['mAP'], savePath=args.savepath, showGraphic=False)
+        return voc_sum
+
+    # EVERY POINT INTERPOLATION:
+    elif args.metrics == 'voc2012' or args.metrics == 'auc':
+        logging.info("Running metric with VOC2012 metric; AUC (Area Under Curve)")
```

Comment:
I'd say "Running metric with VOC2012 metric, using the every point interpolation" instead of "Running metric with VOC2012 metric; AUC (Area Under Curve)"

Reply:
Fixed in https://github.com/rafaelpadilla/review_object_detection_metrics/pull/57/commits/27394185d27d073b2f871dbd7e836ee038c16ad9